packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json

```diff
@@ -441,6 +441,114 @@
                     }
                 }
             },
+            "RunTransactionGethKeychainRequest": {
+                "type": "object",
+                "required": [
+                    "web3SigningCredential",
+                    "transactionConfig"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "web3SigningCredential": {
+                        "$ref": "#/components/schemas/Web3SigningCredential",
+                        "nullable": false
+                    },
+                    "transactionConfig": {
+                        "$ref": "#/components/schemas/QuorumTransactionConfig",
+                        "nullable": false
+                    },
+                    "timeoutMs": {
+                        "type": "number",
+                        "description": "The amount of milliseconds to wait for a transaction receipt with thehash of the transaction(which indicates successful execution) beforegiving up and crashing.",
+                        "minimum": 0,
+                        "default": 60000,
+                        "nullable": false
+                    }
+                }
+            },
+            "RunTransactionGethKeychainResponse": {
+                "type": "object",
+                "required": [
+                    "transactionReceipt"
+                ],
+                "properties": {
+                    "transactionReceipt": {
+                        "$ref": "#/components/schemas/Web3TransactionReceipt"
+                    }
+                }
+            },
+            "RunTransactionPrivateKeyRequest": {
+                "type": "object",
+                "required": [
+                    "web3SigningCredential",
+                    "transactionConfig"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "web3SigningCredential": {
+                        "$ref": "#/components/schemas/Web3SigningCredential",
```

Comment:
@TonyRowntree You could narrow down the type of the signing credential here otherwise people could assume that they can use any one of the possible signing credential types.